docs/source/science/ESA_CCI/ESA_CCI_V4.ipynb

```diff
@@ -17,52 +17,39 @@
     "tags": []
```

Comment:
<p>So if I recap what we're doing, we</p><p> </p><ol><li>Search for STAC items in the ESA CCI collection, keeping only items falling in this bounding box <code>[-55,-6,-54.8,-5.8]</code></li><li>The result is a single item, which covers <code>[-60, -10, -50, 0]</code></li><li>We turn that into a labeled array, from which we subsequently keep only data points in <code>[-55, -5.0, -50, -3]</code></li></ol><p> </p><p>The intent of the notebook is a little confusing because (3) isn't in (1). The notebook looks like we're starting it by saying 'ok let's look for data in area A.' and then in next cells, 'let's look for data in a completely different area B !'. Instead, simple suggestion to stay consistent : in (3), apply (1) again. So, reference only the first bounding box you're creating at the top of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I mapped these three areas so that hopefully what I mean is clearer. The smallest is (1), the largest is (2) and the other is (3).

![Screenshot 2023-09-25 at 11 21 56 AM](https://github.com/MAAP-Project/maap-documentation/assets/45140658/8e3fb7ba-2cc3-4ff5-a62b-13edc72bceec)

Reply:
<p>Thanks for pointing that out. I agree, we should use the same bbox as defined in (1).</p>